validation-test/lit.site.cfg.in

```diff
@@ -125,6 +125,10 @@ if "@SWIFT_BACK_DEPLOY_CONCURRENCY@" == "TRUE":
 if "@SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED@" == "TRUE":
     config.available_features.add('distributed')
 
+config.swift_freestanding_is_darwin = "@SWIFT_FREESTANDING_IS_DARWIN@" == "TRUE"
+config.swift_enable_dispatch = "@SWIFT_ENABLE_DISPATCH@" in ["TRUE", "ON"]
```

Comment:
Same here. This probably should just be normalized.

Reply:
Addressed in fe71878